addons/website_blog/controllers/main.py

```diff
@@ -170,6 +170,14 @@ def tags_list(tag_ids, current_tag):
         response = request.website.render("website_blog.blog_post_short", values)
         return response
 
+    @http.route(['/blog/<model("blog.blog"):blog>/<any("atom","rss"):feed_type>'], type='http', auth="public", website=True)
+    def blog_feed(self, blog, feed_type, **post):
+        limit = int(post.get('limit', 15))
+        cr, uid = request.cr, request.uid
+        base_url = request.registry['ir.config_parameter'].get_param(cr, uid, 'web.base.url')
```

Comment:
add a local function who will be called to render the date field in the template
``` 
        def _blog_date_format(datetime_str, output_format):
            return datetime.datetime.strptime(datetime_str, DEFAULT_SERVER_DATETIME_FORMAT).replace(tzinfo=pytz.utc).strftime(output_format)
```
pass it to the render values.